packages/terra-abstract-modal/src/_ModalContent.jsx

```diff
@@ -5,6 +5,7 @@ import classNames from 'classnames';
 import classNamesBind from 'classnames/bind';
 import ThemeContext from 'terra-theme-context';
 import VisuallyHiddenText from 'terra-visually-hidden-text';
+import FocusLock from 'react-focus-lock';
```

Comment:
What is the difference between react-focus-trap and react-focus-lock?  Don't we usually use the former?

Reply:
As FocusTrap is not working in few of the scenarios like single select dropdown inside modal manger popup. Hence Instead of using FocusTrap now we are using FocusLock which is working in all the scenarios.